### PR TITLE
fix: GPRs for mitochondrial 2-enoyl-CoA hydratase reactions

### PR DESCRIPTION
**Pull request by Devlin-Moyer**
#### Main improvements in this PR:

As proposed in #757:
- Changes GPRs of `MAR01163`, `MAR01177`, `MAR01182`, `MAR01206`, `MAR01219`, `MAR01224`, `MAR01255`, `MAR01272`, `MAR03108`, `MAR03112`, `MAR03116`, `MAR03122`, `MAR03129`, `MAR03171`, `MAR03175`, `MAR03179`, `MAR03183`, `MAR03187`, `MAR03219`, `MAR03223`, `MAR03227`, `MAR03231`, `MAR03235`, `MAR03241`, `MAR03245`, `MAR03252`, `MAR03260`, `MAR03277`, `MAR03281`, `MAR03285`, `MAR03449`, and `MAR03453` to `ENSG00000084754` (HADHA)
- Changes GPRs of `MAR03136`, `MAR03143`, `MAR03150`, `MAR03157`, `MAR03191`, `MAR03195`, `MAR03199`, `MAR03290`, `MAR03523`, and `MAR03528` to `ENSG00000084754 or ENSG00000127884` (HADHA or ECHS1)
- Changes GPRs of `MAR03164`, `MAR03203`, `MAR03532`, `MAR03753`, `MAR03785`, and `MAR04735` to `ENSG00000127884` (ECHS1)

**I hereby confirm that I have:**
- [X] Tested my code on my own computer for running the model
- [X] Selected `develop` as a target branch
- [X] Any removed reactions and metabolites have been moved to the corresponding deprecated identifier lists

### CODE REVIEW

File: model/Human-GEM.yml [hunk 1]
```diff
@@ -119559,7 +119559,7 @@
           - MAM02689m: 1
       - lower_bound: 0
       - upper_bound: 1000
-      - gene_reaction_rule: "ENSG00000084754 and ENSG00000138029 and ENSG00000167315"
```
Comment (JHL-452b) [inline model/Human-GEM.yml:119562]: `MAR03181` has not been mentioned in this [PR](https://github.com/SysBioChalmers/Human-GEM/pull/808) or [issue](https://github.com/SysBioChalmers/Human-GEM/issues/757). The GPR is also different from other reactions. This modification seems inappropriate.

Comment (Devlin-Moyer) [reply model/Human-GEM.yml:119562]: oh yea that is definitely wrong; I think I must've misread `MAR03187` (which is on the list) as `MAR03181`. I will revert that change

**Comment by JHL-452b** [comment]
LGTM!

**Comment by feiranl** [comment]
According to the model yaml file, it seems that grRules for rxn `MAR03281` hasn't been changed as said. It is still `'ENSG00000084754 and ENSG00000113790 and ENSG00000127884'. Please double check this @Devlin-Moyer 
` Thanks!

**Review by feiranl — APPROVED** [review]
LGTM!

**Pull request merged**